src/Compilers/Core/Portable/Compilation/DataFlowAnalysis.cs

```diff
@@ -32,6 +32,15 @@ public abstract class DataFlowAnalysis
         /// </summary>
         public abstract ImmutableArray<ISymbol> DataFlowsOut { get; }
 
+        /// <summary>
+        /// The set of local variables which are definitely assigned a value when a region
+        /// is entered.
+        /// 
+        /// The set of variables that are definitely assigned a value when a region is exited is
```

Comment:
💡 This should either be wrapped in `<para>`, or merged with the previous paragraph. XML whitespace normalization requires that the preceding blank line be ignored during processing, so this comment in its current form has one paragraph with two sentences.

Reply:
resolved.